src/seedcase_sprout/paths.py

```diff
@@ -105,3 +107,15 @@ def resource_batch_files(self, resource_name: str) -> list[Path]:
     def properties_template(self) -> Path:
         """Path to the properties template."""
         return self.root() / "scripts" / "properties.py"
+
+    def resource_properties_template(self, resource_name: str = "") -> Path:
```

Comment:
```suggestion
    def resource_properties_script(self, resource_name: str = "") -> Path:
```

Since it isn't the path to the template, its the path to the newly created file.